### ISSUE TITLE
open viewer in different view, not as an editor

### ISSUE THREAD

**Issue by tolot27** [Borisvl/Pdf4Eclipse#13]
Currently, the viewer is opened just as another tab in the main editor. It can not attached to any other views like the one hosting the console or tasks.
I can only moved to another side of the editor and then appears separately. Unfortunately, every new tex document is opened then as a tab next to the pdfviewer tab and not in the main editor.

**Comment by yngwi** [comment]
I would also like this feature very much! It would be most practical.

**Comment by Borisvl** [comment]
This is a restriction of Eclipse that editors (and Pdf4Eclipse is an editor) can only be opened in the editor pane. In the e4 version of Eclipse this behavior was changed and so this should not be an issue anymore when the e4 version will be officially released.

**Comment by tolot27** [comment]
Then way is Pdf4Eclipse an editor at all? Sounds a bit strange to me to implement a viewer as an editor.

**Comment by Borisvl** [comment]
Yes, indeed it looks rather strange. But it is has a lot of advantages like opening more than one pdf at once, the outline view, and the connection to the file extension. Implementing the reader in a view would make it much more harder to implement these aspects.
